frontend/src/scenes/dashboard/DashboardEditBar.tsx

```diff
@@ -16,7 +17,14 @@ export function DashboardEditBar(): JSX.Element {
     const disabledReason = !canEditDashboard ? "You don't have permission to edit this dashboard" : undefined
 
     return (
-        <div className="flex gap-2 items-center justify-between flex-wrap">
+        <div
+            className={clsx(
+                'flex gap-2 items-center justify-between flex-wrap border',
+                dashboardMode === DashboardMode.Edit
+                    ? '-m-1.5 p-1.5 border-border-bold border-dashed rounded-lg'
+                    : 'border-transparent'
+            )}
+        >
```

Comment:
Just making sure since we removed this previously: is this addition of the outline intended?

Reply:
Yes, this is for the scenario when user changes filters then clicks into an insight and then navigates back to the dashboard.
At this stage the dashboard is in edit mode, but want to highlight the dates/filters so the user knows what's changed in edit mode if they hit save

<img width="839" alt="image" src="https://github.com/user-attachments/assets/43384e54-9305-4e91-96f2-83cbb4f25a7b">